src/Grid/FeatureGrid/FeatureGrid.tsx

```diff
@@ -41,11 +62,48 @@ const rowKeyClassNamePrefix = 'row-key-';
 
 const cellRowHoverClassName = 'ant-table-cell-row-hover';
 
+const DragIndexContext = createContext<DragIndexState>({ active: -1, over: -1 });
+
+const dragActiveStyle = (dragState: DragIndexState, id: string) => {
+  const { active, over, direction } = dragState;
+  // drag active style
+  let style: React.CSSProperties = {};
+  if (active && active === id) {
+    style = { backgroundColor: 'gray', opacity: 0.5 };
```

Comment:
Not sure if possible, but there are several inline css variations. Can we move them to the appropriate `*.less` files?

Reply:
I think that might be possible but the component would need some refactoring. I could address that, aswell as the <th></th> return in a follow-up.